akka-cluster/src/main/resources/reference.conf

```diff
@@ -152,6 +152,11 @@ akka {
     # longer than this it could lead to removed nodes being re-added to the cluster
     prune-gossip-tombstones-after = 24h
 
+    # Try to limit the number of connections between datacenters used for gossip and heartbeating
+    # to this number. This will not limit connections created for the messaging of the application.
+    # If the cluster is not multi dc this value is ignored.
```

Comment:
I'd rather say "If the cluster does not span multiple data centers, this value has no effect"?